File: Python/p11936.py
kasus = int(input())
while kasus > 0:
    masukan = input()
    masukan2 = masukan.split()
    num1 = int(masukan2[0])
    num2 = int(masukan2[1])
    num3 = int(masukan2[2])

    if num1+num2 > num3 and num1+num3 > num2 and num2+num3 > num1:
        print("OK")
    else:
        print("Wrong!!")

    kasus -= 1

File: Java/p11936.java
import java.util.Scanner;

public class p11936 {
    public static void main (String [] args) {
        Scanner input = new Scanner(System.in);
        int kasus = input.nextInt();
        for (int i = 0; i < kasus; i++) {
            int num1 = input.nextInt();
            int num2 = input.nextInt();
            int num3 = input.nextInt();

            if (num1+num2 > num3 && num1+num3 > num2 && num2+num3 > num1) {
                System.out.println("OK");
            } else {
                System.out.println("Wrong!!");
            }
        }
    }
}